unitary/engine_utils.py

```diff
@@ -247,13 +247,7 @@ def engine(self):
         return self._engine
 
     def get_sampler(self, gateset: str = None):
-        if gateset == 'sycamore':
-            gateset = cg.SYC_GATESET
-        elif gateset == 'sqrt-iswap':
-            gateset = cg.SQRT_ISWAP_GATESET
-        else:
-            raise ValueError("Unknown gateset {}".format(gateset))
-        return self.engine.sampler(processor_id=self.processor_id, gate_set=gateset)
+        return self.engine.sampler(processor_id=self.processor_id)
```

Comment:
What's this part? Doesn't seem related to type checking.

Reply:
gateset parameter no longer exists in Cirq 1.0.  Added to the PR description.

I can split it out into a different PR if you want.

Reply:
That's all right. A different commit would have been nice though.